cx-portal/src/components/pages/AppDetail/components/AppDetailHeader/index.tsx

```diff
@@ -88,7 +88,8 @@ export default function AppDetailHeader({ item }: AppDetailHeaderProps) {
       default:
         btnColor = {
           color:
-            user.roles.indexOf('subscribe_apps') !== -1
+            user.roles.indexOf('subscribe_apps') !== -1 &&
```

Comment:
please use enum type for this and following hard coded strings

Reply:
@oyo Made the changes. Please review again